open_spiel/games/phantom_go.h

```diff
@@ -0,0 +1,171 @@
+// Copyright 2019 DeepMind Technologies Ltd. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef OPEN_SPIEL_GAMES_PHANTOM_GO_H_
+#define OPEN_SPIEL_GAMES_PHANTOM_GO_H_
+
+#include <array>
+#include <cstring>
+#include <map>
+#include <memory>
+#include <string>
+#include <unordered_set>
+#include <vector>
+
+#include "open_spiel/games/phantom_go/phantom_go_board.h"
+#include "open_spiel/spiel.h"
+#include "open_spiel/spiel_utils.h"
+
+// Game of Go:
+// https://en.wikipedia.org/wiki/Go_(game)
+//
+// Parameters:
+//  "komi"        float  compensation for white                  (default = 7.5)
+//  "board_size"  int    rows of the board, usually 9, 13 or 19  (default = 19)
+//  "handicap"    int    number of handicap stones for black     (default = 0)
```

Comment:
max_game_length is missing

Reply:
This comment was already missing in the Go implementation, so I added it. I also fixed the comment few lines up on line 30, changed it to phantom go and changed the link to a site about phantom go.